airflow/io/store/__init__.py

```diff
@@ -0,0 +1,125 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, ClassVar
+
+from airflow.io.fsspec import SCHEME_TO_FS
+from airflow.utils.module_loading import import_string, qualname
+
+if TYPE_CHECKING:
+    from fsspec import AbstractFileSystem
+
+
+class Store:
```

Comment:
Is this also an ABC missing the base class reference? Assuming there is an `ObjectStore` providing concrete implementations?

Reply:
`Store` is concrete. I don't think there is a need for abstraction